src/development/ui/animations/tutorial.md

```diff
@@ -204,8 +204,8 @@ colorTween = ColorTween(begin: Colors.transparent, end: Colors.black54);
 ```
 
 A `Tween` object does not store any state. Instead, it provides the
-`evaluate(Animation<double> animation)` method that
-applies the mapping function to the current value of the animation.
+[`Animation.value`][] method that applies the mapping function 
+to the current value of the animation.
```

Comment:
**question:** What do we mean by mapping function? Does the `Tween` object's value map to an `Animation.value` at a given point in time? If so, that would be the more clear way to say this.

Reply:
cc @HansMuller, I think that Tony's statement is correct, but can you confirm?